epoxy-adapter/src/main/java/com/airbnb/epoxy/EpoxyVisibilityTracker.java

```diff
@@ -169,12 +214,21 @@ private void processChild(@NonNull View child, boolean detachEvent, String event
     if (recyclerView != null) {
       final ViewHolder holder = recyclerView.getChildViewHolder(child);
       if (holder instanceof EpoxyViewHolder) {
-        processVisibilityEvents(
+        boolean changed = processVisibilityEvents(
             recyclerView,
             (EpoxyViewHolder) holder,
             detachEvent,
             eventOriginForDebug
         );
+        if (changed) {
+          if (child instanceof EpoxyRecyclerView) {
```

Comment:
we are supporting any Recyclerview if they manually attach, right? should the `child instanceof EpoxyRecyclerView` check be removed?